controllers/consoleplugin/consoleplugin_objects.go

```diff
@@ -84,6 +85,39 @@ func (b *builder) consolePlugin() *osv1alpha1.ConsolePlugin {
 	}
 }
 
+func (b *builder) consolePluginServiceMonitor() *monitoringv1.ServiceMonitor {
+	return &monitoringv1.ServiceMonitor{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      constants.PluginServiceMonitorName,
+			Namespace: constants.DefaultOperatorNamespace,
+		},
+		Spec: monitoringv1.ServiceMonitorSpec{
+			Endpoints: []monitoringv1.Endpoint{
+				{
+					Port:     "main",
+					Interval: "15s",
+					Scheme:   "https",
+					TLSConfig: &monitoringv1.TLSConfig{
+						SafeTLSConfig: monitoringv1.SafeTLSConfig{
+							InsecureSkipVerify: true,
```

Comment:
@jpinsonneau I'm not sure what I need to do here. Do I need to duplicate the configMap or just provide `CertFile: "/var/serving-cert"` ?

Reply:
You can't use `"/var/serving-cert"` here since it's only mounted in `ConsolePlugin` `Pod`.

`SafeTLSConfig` seems to be able to load `Cert` from secret:
```go
SafeTLSConfig: monitoringv1.SafeTLSConfig{
	Cert: monitoringv1.SecretOrConfigMap{
		Secret: &corev1.SecretKeySelector{
			LocalObjectReference: corev1.LocalObjectReference{
				Name: console-serving-cert
			},
			Key: "tls.crt",
		},
	},
},
```

Have you tried that ?

Reply:
done